main.cpp

```diff
@@ -1022,15 +1025,19 @@ static void spawn_watchman(void) {
   }
 #endif
 
+  if (no_system_spawner) {
```

Comment:
I think you'll need to enclose this inside `#ifndef _WIN32`

Reply:
indeed - should be fixed now.